baseplate/integration/wrapped_context.py

```diff
@@ -0,0 +1,21 @@
+import copy
+
+
+def merge_two_dicts(x, y):
+    """Given two dicts, merge them into a new dict as a shallow copy."""
+    z = x.copy()
+    z.update(y)
+    return z
+
+
+class WrappedRequestContext(object):
+    def __init__(self, context, trace=None):
+        self._context = context
+        self.__dict__.update(self._context.__dict__)
```

Comment:
This was mainly to support passing in the wrapped context to existing framework integrations (which make some direct calls that were troublesome). I think I understand enough of how all of this is working now to no longer need this + move to only having wrapped contexts in baseplate components.